docs/source/item-validation.rst

```diff
@@ -193,6 +221,38 @@ as a `dict`:
         OtherItem: '/path/to/otheritem_schema.json',
     }
 
+.. _SPIDERMON_VALIDATION_CERBERUS:
+
+SPIDERMON_VALIDATION_CERBERUS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Default: ``None``
+
+A `list` containing the location of the item schema. Could be a local path, URL or the schema itself.
+
+.. code-block:: python
+
+    # settings.py
+
+    SPIDERMON_VALIDATION_CERBERUS = [
+        '/path/to/schema.json',
+        'http://example.com/mycerberusschema',
+        {"Field": {"type": "number", "required":True}}
+    ]
```

Comment:
I assume that, when multiple schemas are defined, an item is considered valid if it passes *any* of these schemas. I think this should be clarified in the documentation.

Reply:
When multiple schemas (2 where I tested) are defined, the item is to be considered valid when it passes through both schemas. To clarify further here's are the stats. As it still detects errors in the quotes field. 

```
{'downloader/request_bytes': 446,
 'downloader/request_count': 2,
 'downloader/request_method_count/GET': 2,
 'downloader/response_bytes': 2701,
 'downloader/response_count': 2,
 'downloader/response_status_count/200': 1,
 'downloader/response_status_count/404': 1,
 'elapsed_time_seconds': 1.898199,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2019, 8, 30, 18, 8, 58, 13614),
 'item_scraped_count': 10,
 'log_count/DEBUG': 12,
 'log_count/INFO': 10,
 'memusage/max': 61636608,
 'memusage/startup': 61636608,
 'response_received_count': 2,
 'robotstxt/request_count': 1,
 'robotstxt/response_count': 1,
 'robotstxt/response_status_count/404': 1,
 'scheduler/dequeued': 1,
 'scheduler/dequeued/memory': 1,
 'scheduler/enqueued': 1,
 'scheduler/enqueued/memory': 1,
 'spidermon/validation/fields': 10,
 'spidermon/validation/fields/errors': 20,
 'spidermon/validation/fields/errors/invalid_int': 10,
 'spidermon/validation/fields/errors/invalid_int/quote': 10,
 'spidermon/validation/fields/errors/invalid_number': 10,
 'spidermon/validation/fields/errors/invalid_number/quote': 10,
 'spidermon/validation/items': 10,
 'spidermon/validation/items/errors': 20,
 'spidermon/validation/validators': 2,
 'spidermon/validation/validators/item/cerberus': True,
 'start_time': datetime.datetime(2019, 8, 30, 18, 8, 56, 115415)}
```

Reply:
Let me know if I should move forward with adding a clarification.

Reply:
Roger that. Coming up now!